destinationCreation.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+      <meta charset="UTF-8"/>
+      <title>Destination Creation</title>
+      <link rel="stylesheet" href="destinationCreation.css"/>
+      <script src="destinationCreation.js"></script>
```

Comment:
Does this file exist yet? Maybe it does and github's diff view is just hiding it. But if not, could you either create the file as just empty or a blank line, or remove this `script` tag?